apps/src/applab/dropletConfig.js

```diff
@@ -17,7 +17,9 @@ var DEFAULT_HEIGHT = (480 - applabConstants.FOOTER_HEIGHT).toString();
 
 // Flip the argument order so we can bind `typeFilter`.
 function chooseAsset(typeFilter, callback) {
-  dashboard.assets.showAssetManager(callback, typeFilter, null, !studioApp.reduxStore.getState().pageConstants.is13Plus);
+  dashboard.assets.showAssetManager(callback, typeFilter, null, {
+    showUnderageWarning: !studioApp.reduxStore.getState().pageConstants.is13Plus
+  });
```

Comment:
I think this will need to be tackled separately since I don't fully understand why some apps behave differently than others and I don't want to change something that is related to COPPA policy as a side effect of cleaning up the code.